src/components/MyAccount/CheckoutsTab/RenewButton.tsx

```diff
@@ -35,15 +35,20 @@ const RenewButton = ({
     ),
     closeButtonLabel: "OK",
     headingText: (
-      <Box className={styles.modalHeading}>
-        <Icon
-          size="large"
-          name="actionCheckCircleFilled"
-          color="ui.success.primary"
-        />
-        <Text sx={{ marginBottom: 0 }}> Renewal successful </Text>
-      </Box>
+      <Heading className={styles.modalHeading}>
+        <>
+          <Icon
+            size="large"
+            name="actionCheckCircleFilled"
+            color="ui.success.primary"
+          />
+          <Text sx={{ marginBottom: 0 }}> Renewal successful </Text>
+        </>
+      </Heading>
     ),
+    onClose: () => {
```

Comment:
Out of curiosity, does the shorthand work? Or `onClose: onClose,`?

Reply:
<img width="963" alt="Screenshot 2024-05-16 at 2 27 57 PM" src="https://github.com/NYPL/research-catalog/assets/73774046/ccfcccbb-f61b-4a7d-b6be-870d56d6a969">
no which is unfortunate

Reply:
Try without invoking the function `onClose: onClose` instead of `onClose: onClose()`. Perhaps in #213